input/fsh/SD_Extensions.fsh

```diff
@@ -333,3 +333,32 @@ Expression: "extension.where(url = 'energy').exists() implies \n
                 )\n
               )"
 XPath: "true()"
+
+Invariant:   TG263BED
+Description: "Unit must match TG263 BED Regexp"
+Expression:  "$this.toString().matches('(CGE$)|(BED|EQD2Gy)([(]a\\/b=[0-9]*.?[0-9]*[)])?$')"
+Severity:    #warning
+
+Extension: BiologicallyEffectiveDose
+Id: codexrt-radiotherapy-biologically-effective-dose
+Title: "Biologically Effective Dose Extension"
+Description: "Extension providing biologically effective dose.  The following forms are supported:
+       CGE  - Cobalt Gray Equivalent 
+       BED  - Biological Effective Dose
+       BED(a/b=<decimal>) - BED with alpha/beta
+       EQD2Gy -  Equivalent Dose in 2 Gy per Fraction 
+       EQD2Gy(a/b=<decimal>) - EQD2Gy with alpha/beta
+
+The units are assumed to by Gy.
```

Comment:
typo "by Gy" -> "be Gy". But I would rather not constrain the unit here. Maybe say "for the above forms"?

Reply:
This was from Chuck, so I'll leave it for now.

Reply:
Added RBE.  Need to beef up the regexp to support RBE(filter = <decimal>) and improve documentation.